include/core/cfdp_core/pdu_directive.hpp

```diff
@@ -9,12 +9,12 @@ using ::cfdp::pdu::header::LargeFileFlag;
 namespace cfdp::pdu::directive
 {
 
-class KeepAlivePdu : PduInterface
+class KeepAlive : PduInterface
 {
   public:
-    KeepAlivePdu(uint32_t progress) : progress(progress), largeFileFlag(LargeFileFlag::SmallFile){};
-    KeepAlivePdu(uint64_t progress) : progress(progress), largeFileFlag(LargeFileFlag::LargeFile){};
-    KeepAlivePdu(std::span<uint8_t const> memory);
+    KeepAlive(uint32_t progress) : progress(progress), largeFileFlag(LargeFileFlag::SmallFile){};
```

Comment:
Yeah, the same thing here - those two constructors are very weird

Reply:
SAA

Reply:
agree